File: Skattata.Core/SieVoucherRow.cs
namespace Skattata.Core;

public class SieVoucherRow : SieDataItem
{
    public SieVoucherRow()
    {
        Objects = new List<SieObject>();
    }

    public string AccountNumber { get; set; } = "";
    public List<SieObject> Objects { get; set; }
    public decimal Amount { get; set; }
    public DateTime TransactionDate { get; set; }
    public string RowText { get; set; } = "";
    public decimal Quantity { get; set; }
    public string RegistrationSign { get; set; } = "";
}

File: packages/sie-core/src/models/SieVoucherRow.ts
import { SieObject } from './SieObject.js';

export class SieVoucherRow {
  accountNumber: string = '';
  objects: SieObject[] = [];
  amount: number = 0;
  transactionDate: Date = new Date(0);
  rowText: string = '';
  quantity: number = 0;
  registrationSign: string = '';
}
